packages/shared/src/lib/featureManagement.ts

```diff
@@ -42,6 +42,7 @@ const feature = {
   ),
   extensionOverlay: new Feature('onboarding_extension_overlay', false),
   seniorContentOnboarding: new Feature('senior_content_onboarding', false),
+  plusSubscription: new Feature('plus_subscription', false),
```

Comment:
`/plus` page also behind experiment so someone does not accidentally go there?

Reply:
Easter egg? 👼

Reply:
We'll just show the page.
https://dailydotdev.slack.com/archives/C07V6888KML/p1731598587592599